GitHub integration: FE support for creating new PAT and Enterprise accounts

estib-vega opened the pull request:
Allow the user to store their PAT and determine a host for their GitHub integration.

estib-vega committed:
GH integration: Display nice pills for the type

Display the type as a pill next to the user name for a given account

estib-vega committed:
GitHub Integration: Use the Host in the client

When creating the client, pass the GitHub Enterprise host if set

PavelLaptev committed:
fix context menu binding

PavelLaptev committed:
update icons for the context menu

PavelLaptev commented:
https://github.com/user-attachments/assets/3226d242-f4da-451f-bcb4-0e8e8b2c25dc

---

Designs in Figma here: https://www.figma.com/design/ShIiR6hI5dzH7sT5L03Jg1/App-Design-3.5?node-id=15325-127586&t=jQEiai92ofKeTDdc-1

PavelLaptev committed:
update layout and handle errors

estib-vega committed:
GitHub integration: PAT and GitHub enterprise support

Add the option to add a GitHub Personal Acess token and optionally the GitHub Enterprise host as credentials for the integration

Also: disable Add Account button while a flow is showing to prevent interactions (added disabled binding).

estib-vega commented:
> @estib-vega I forgot to check selectors in the project settings, sorry. Can I check it before we merge?

Sure! no problem
